src/GongSolutions.WPF.DragDrop.Shared/DragDrop.cs

```diff
@@ -146,21 +146,21 @@ private static void CreateEffectAdorner(DropInfo dropInfo)
       }
     }
 
-    private static DataTemplate GetEffectAdornerTemplate(UIElement target, DragDropEffects effect, string destinationText)
+    private static DataTemplate GetEffectAdornerTemplate(UIElement target, DragDropEffects effect, string destinationText, string effectText = null)
     {
       switch (effect)
       {
         case DragDropEffects.All:
           // TODO: Add default template for EffectAll
           return GetEffectAllAdornerTemplate(target);
         case DragDropEffects.Copy:
-          return GetEffectCopyAdornerTemplate(target) ?? CreateDefaultEffectDataTemplate(target, IconFactory.EffectCopy, "Copy to", destinationText);
+          return GetEffectCopyAdornerTemplate(target) ?? CreateDefaultEffectDataTemplate(target, IconFactory.EffectCopy, effectText == null ? "Copy to" : effectText, destinationText);
```

Comment:
So maybe it's also good to check `string.IsNullOrEmpty(effectText)`

Reply:
We used to have the code like your suggestion but we needed an empty string instead of "move to".
With string.IsNullOrEmpty(effectText) the empty string would have been replaced with "Move to". A workaround would have been to set the effectText to " " but the additional whitespace doesn't look good.

Reply:
You're right, so it's ok for me.